src/timely-util/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "timely-util"
+description = "Utilities for working with Timely."
+version = "0.0.0"
+edition = "2021"
+publish = false
+
+[dependencies]
+futures = { version = "0.3.19", optional = true }
+futures-util = "0.3.19"
+timely = { git = "https://github.com/TimelyDataflow/timely-dataflow", default-features = false, features = ["bincode"] }
+tokio = { version = "1.15.0", features = ["io-util", "macros", "net", "rt-multi-thread", "time"] }
```

Comment:
tokio is only used in the tests so we can move it under dev-dependencies. Also I think we only require the `macros` and `time` features